lib/autoyast.pm

```diff
@@ -802,7 +802,7 @@ sub generate_json_profile {
     my $profile_name = "generated_profile.json";
     my $profile_path = get_required_var('CASEDIR') . "/data/" . $profile;
 
-    my @profile_options = map { "--tla-" . (/true|false/ ? "code" : "str") . " $_ " }
+    my @profile_options = map { "--tla-" . (/true|false|=[0-9]/ ? "code" : "str") . " $_ " }
```

Comment:
please separate both variables, they are not connected on purpose, I asked developers a time ago why they didn't reuse it, it is more clear this way even for our our automation code.